src/System.Runtime.CompilerServices.Unsafe/src/include/netfx/coreassembly.h

```diff
@@ -0,0 +1,8 @@
+#define CORE_ASSEMBLY "mscorlib"
+
+// Metadata version: v4.0.30319
+.assembly extern CORE_ASSEMBLY
+{
+  .publickeytoken = (B7 7A 5C 56 19 34 E0 89 )                         // .?_....:
```

Comment:
@ericstj I'm not too familiar with ildasm, so I'm not 100% sure about these changes but everything worked fine locally and the produced assembly looked right to me. Do you know who is the expert so they can review this?

Reply:
Looks ok, the comment appears wrong. 😆 

You can delete it if you want, if you want to correct it, this is what ILDasm would normally emit:
```
  .publickeytoken = (B7 7A 5C 56 19 34 E0 89 )                         // .z\V.4..
```

Reply:
I'll fix this.